src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1744,17 +1758,23 @@ def _aks_browse(
         logger.warning('Proxy running on %s', proxy_url)
 
     timeout = None
-    if os.getenv("PYTEST_CURRENT_TEST", None):
+    test_hook_data = get_cmd_test_hook_data("test_aks_browse_legacy_hook.json")
+    if test_hook_data and test_hook_data == "enabled":
```

Comment:
please give a struct to this .json.

Reply:
Sure, changed in commit [0f6de86](https://github.com/Azure/azure-cli/pull/19115/commits/0f6de86183629cd7eb5e358af52cbf4ee8a17016).